tree/dataframe/test/dataframe_interface.cxx

```diff
@@ -280,12 +280,12 @@ TEST(RDataFrameInterface, InvalidDefine)
    try {
       df.Define("1", [] { return true; });
    } catch (const std::runtime_error &e) {
-      EXPECT_STREQ("Cannot define column \"1\": not a valid C++ variable name.", e.what());
+      EXPECT_STREQ("RDataFrame::Define: cannot define column \"1\". Not a valid C++ variable name.", e.what());
```

Comment:
Do we have a similar test for the `Alias` case?

Reply:
around lines 60 in this file we have similar tests for Alias. I'll add one for the case in which the alias name is not a valid C++ variable name, I think that's missing

Reply:
done

Reply:
Go go go! :)